src/daos-deserialize/daos-deserialize.c

```diff
@@ -0,0 +1,225 @@
+#include <stdio.h>
+#include <getopt.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <errno.h>
+#include <stdbool.h>
+
+/* for daos */
+#ifdef DAOS_SUPPORT
+#include "mfu_daos.h"
+#endif
+#include "mpi.h"
+#include "mfu.h"
+
+#include "mfu_errors.h"
+
+/** Print a usage message. */
+void print_usage(void)
+{
+    printf("\n");
+    printf("Usage: daos-serialize [options] <h5file> <h5file> ... \n");
+    printf("\n");
+    printf("Options:\n");
+    printf("  -p, --pool               - pool uuid for containers\n");
+    printf("  -v, --verbose            - verbose output\n");
+    printf("  -h, --help               - print usage\n");
+    printf("For more information see https://mpifileutils.readthedocs.io.\n");
+    printf("\n");
+    fflush(stdout);
+}
+
+int main(int argc, char** argv)
+{
+    /* assume we'll exit with success */
+    int rc = 0;
+
+    /* initialize MPI */
+    MPI_Init(&argc, &argv);
+    mfu_init();
+
+    /* get our rank */
+    int rank;
+    MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+    /* verbose by default */
+    mfu_debug_level = MFU_LOG_VERBOSE;
```

Comment:
@dsikich Similar to my comment for `daos-serialize`, maybe we want to add a quiet option